bin/sequencer

```diff
@@ -158,6 +158,6 @@ fi
 fgrep RESULT $site_model/out/devices/$device_id/RESULT.log | \
     fgrep ' schemas ' | sort -k 3 | tee $SCHEMA_OUT
 
-(source venv/bin/activate && bin/sequencer_report $site_model $device_id)
+(source venv/bin/activate && python3 bin/sequencer_report.py $site_model $device_id)
```

Comment:
rather than make them explicitly python, I think a better solution would be to do something like

PATH=venv/bin:$PATH bin/sequencer_report $site_model $device_id -- so it will pick up the venv as needed.